kubernetes-gcp-python/__main__.py

```diff
@@ -0,0 +1,117 @@
+import pulumi
+import pulumi_gcp as gcp
+
+# Get some provider-namespaced configuration values
+provider_cfg = pulumi.Config("gcp")
+gcp_project = provider_cfg.require("project")
+gcp_region = provider_cfg.get("region")
+if gcp_region is None:
+    gcp_region = "us-central1"
```

Comment:
Should we use Laura's default-value thing here, and with the others?

```suggestion
gcp_region = provider_cfg.get("region", "us-central1")
```

Reply:
The use of .get is significantly neater.

Reply:
Good catch! Fixed in 440fb15.